apps/mull-ui/src/app/components/chat-bubble-list/chat-bubble-list.tsx

```diff
@@ -31,7 +32,11 @@ export const ChatBubbleList = ({ posts, subToMore }: ChatBubbleListProps) => {
         chatDate={formatChatBubbleDate(new Date(post.createdTime))}
         userPicture={avatarUrl(post.user)}
       >
-        {post.message}
+        <>
+          {post.medias && <img src={getMediaUrl(post.medias.id)} className="img" alt="" />}
+          {post.medias && <br />}
```

Comment:
💡
1. It would be better to not use `br`, and instead make the `img` element display block.
2. If you insist on keeping the `br`, you can just have one if:
```tsx
{post.medias && 
<>
  <img src={getMediaUrl(post.medias.id)} className="img" alt="" />
  <br />
</>
}
```